pkg/verifier/notation/notation.go

```diff
@@ -70,8 +72,9 @@ func init() {
 	factory.Register(verifierName, &notationPluginVerifierFactory{})
 }
 
-func (f *notationPluginVerifierFactory) Create(_ string, verifierConfig config.VerifierConfig, pluginDirectory string) (verifier.ReferenceVerifier, error) {
-	conf, err := parseVerifierConfig(verifierConfig)
+func (f *notationPluginVerifierFactory) Create(_ string, verifierConfig config.VerifierConfig, pluginDirectory string, namespace string) (verifier.ReferenceVerifier, error) {
+	logrus.Debugf("notation create with config %v, namespace '%v'", verifierConfig, namespace)
```

Comment:
I think we should be using the new context based logger instead of logrus directly

Reply:
refer to https://ratify.dev/docs/1.0/reference/logger

Reply:
fixed